test/screenshot/lib/cli-arg-parser.js

```diff
@@ -19,7 +19,11 @@
 const GitRepo = require('./git-repo');
 const argparse = require('argparse');
 const fs = require('mz/fs');
-const path = require('path');
+
```

Comment:
Nit: Make these all inline to follow the style of the other imports.

Reply:
Done